packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -39,6 +39,11 @@ import { getXoaPlan, SOURCES } from '../../common/xoa-plans'
 
 const _editProxy = (value, { name, proxy }) => editProxyAppliance(proxy, { [name]: value })
 
+const _editProxyAddress = (value, props) => {
+  value = value.trim()
```

Comment:
Is there a reason to do the trim and the check of the value in `_editProxyAddress` instead of `_editProxy`?